protected/modules/OphCiExamination/migrations/m171011_023359_patient_systemic_diagnosis_view.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+class m171011_023359_patient_systemic_diagnosis_view extends CDbMigration
+{
+    public function createView($view_name, $select)
+    {
+        $this->dbConnection->createCommand('create view ' . $view_name . ' as ' . $select)->execute();
```

Comment:
OEMigration might already have this ability

Reply:
I don't believe it does. I've seen other migrations that have this function defined in them as well.